client/src/pages/Feedback/index.tsx

```diff
@@ -25,7 +25,7 @@ function Feedback() {
           </div>
         </a>
         <a
-          href="https://t.me/+FdDUmLsW8jI0YmUx"
+          href="https://forms.gle/j4XXJ114kftvS5e7A"
```

Comment:
I think you meant to update the form below, not telegram

Reply:
i'm glad you questioned it, I was looking at it and was kind of confused...  but i saw a black uniform code ninja on Github and I assumed it was okay :P